.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+dist: trusty
+addons:
+  firefox: 'latest'
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+
+env:
+  - TEST_SCRIPT=test
+  - TEST_SCRIPT=test:integration
+
+language: node_js
+node_js:
+  - "6"
```

Comment:
Also need 8 since we support all active LTS releases.

https://github.com/nodejs/Release
https://www.polymer-project.org/2.0/docs/tools/node-support#what-versions-of-node.js-do-we-support

Reply:
We support it, but we so far consider running the oldest and the latest node versions to be good enough for testing the CLI.